docs/connector-development/config-based/source_schema.yaml

```diff
@@ -64,42 +60,41 @@ definitions:
     type: object
     anyOf:
       - "$ref": "#/definitions/SimpleRetriever"
-  SimpleRetriever:
+  DefaultRetriever:
     type: object
-    additionalProperties: true
+    additionalProperties: false
     required:
-      - name
       - requester
       - record_selector
     properties:
-      "$options":
-        "$ref": "#/definitions/$options"
-      name:
-        type: string
+      "$parameters":
+        "$ref": "#/definitions/$parameters"
       primary_key:
         "$ref": "#/definitions/PrimaryKey"
       requester:
         "$ref": "#/definitions/Requester"
       record_selector:
-        "$ref": "#/definitions/HttpSelector"
+        "$ref": "#/definitions/RecordSelector"
       paginator:
         "$ref": "#/definitions/Paginator"
-      stream_slicer:
+      iterators:
         "$ref": "#/definitions/StreamSlicer"
+      cursor:
+        "$ref": "#/definitions/CursorStreamSlicer"
```

Comment:
thoughts on renaming DatetimeStreamSlicer? It might be easier to think of it as the cursor